graphql/schema.graphql

```diff
@@ -85,69 +128,121 @@ type CoursePreference {
 }
 
 input CoursePreferenceInput {
-  """Course subject, e.g. SENG, CSC"""
+  """
+  Course subject, e.g. SENG, CSC
+  """
   subject: String!
-  """Course code, e.g. 499, 310"""
+  """
+  Course code, e.g. 499, 310
+  """
   code: String!
-  """Term course is offered in"""
+  """
+  Term course is offered in
+  """
   term: Term!
   preference: Int!
 }
 
 input GenerateScheduleInput {
   year: Int!
+  term: Term!
+  courses: CourseID!
 }
 
 type CourseID {
-  """Course subject, e.g. SENG, CSC"""
+  """
```

Comment:
based on recent meetings, do we need to add `section: String!` to this (i.e. A01, A02)

Reply:
https://github.com/SENG499-S22-Company3/shared/pull/14#discussion_r914034562